scripts/set-more-info-link.py

```diff
@@ -86,14 +85,10 @@ def set_link(file, link):
         locale = "en"
 
     # build new line
-    if locale == "hi":
+    if locale in labels:
         new_line = f"> {labels[locale]} <{link}>।\n"
-    elif locale == "ja" or locale == "th":
-        new_line = f"> {labels[locale]} <{link}>\n"
-    elif locale == "zh" or locale == "zh_TW":
-        new_line = f"> {labels[locale]}<{link}>.\n"
```

Comment:
These are special cases for some languages. You can't just delete them.

Reply:
I'm confused as all those locales show up with valid key-values in the labels dict and they all build the same f-string. Are they supposed to build out different f-strings?

Reply:
f-strings on lines 90 and 92 are identical, the only difference with them and the last to is a period and a space. Neither of which I see are needed to be different.

Reply:
They are different.
`ja` and `th` don't use `.`.
`zh` and `zh_TW` use full width `：` which doesn't need space between `：` and `<{link}>`.

Reply:
Okay I revert that to keep with that interesting styling choice

Reply:
> interesting styling choice

These aren't styling choices, it's part of localization! Different languages, and especially non-Latin languages, can use entirely different punctuation.

For example, in Hindi, they use `।` as a period. That's not style, that's just how it's meant to be localized.